.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,56 @@
+name: coverity-scan
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Weekly at 00:00 UTC
+
+jobs:
+  latest:
+    runs-on: ubuntu-18.04
```

Comment:
Why not use ubuntu-20.04 (latest LTS)?

Reply:
Every build is still on 18.04. I don't mind moving, but would rather do it for all builds in one separate issue.
This way the builds are still lined up.